wm/desk.go

```diff
@@ -32,17 +32,24 @@ type x11WM struct {
 	moveResizingType  moveResizeType
 	altTabList        []desktop.Window
 	altTabIndex       int
-	screens           []*desktop.Head
-	active            *desktop.Head
-	primary           *desktop.Head
+	screens           []*desktop.Screen
```

Comment:
Don't you think that the state for a screens provider should be in the screens provider?
The instance below is only needed because you haven't encapsulated the data appropriately.

Reply:
yes